test/apiv2/80-kube.at

```diff
@@ -0,0 +1,50 @@
+# -*- sh -*-
+#
+# test more container-related endpoints
+#
+
+podman create $IMAGE true
```

Comment:
@edsantiago, can you take a look at this new tests?

We moved over some other kube test as well.  It seemed a valid reasons to create a dedicated test.  The motivation is to avoid silent regressions since `podman-remote` calls the kube/play endpoint now.